src/winforms/toga_winforms/app.py

```diff
@@ -21,8 +21,13 @@ def __init__(self, interface):
     def create(self):
         self.native = WinForms.Application
 
-        if win_version >= 6:
+        if win_version >= 6 and win_version < 8:
             user32.SetProcessDPIAware(True)
+        elif win_version >= 8 and win_build < 15063:
```

Comment:
Are you able to provide any reference to documentation for why these magic numbers exist? An inline docstring will help explain the magic to whoever comes along later.

Reply:
Actually - this specific check looks a little off - a windows 8 build with a high build number will break on this one.

Reply:
You are correct that this check is wrong as the windows 8 major version number is actually a 6 and not an 8. In regards to build number it should only pull the update build number, which for Windows 8 the highest build number is 9600. I will fix this tomorrow when I have the chance.  Source for information: https://www.lifewire.com/windows-version-numbers-2625171

Reply:
I went ahead and implemented much better checks and loads of comments documenting what is what and why.